src/Components/Navbar/Navbar.css

```diff
@@ -0,0 +1,123 @@
+.navbar {
+    display: grid;
+    grid-template-columns: 2fr 3fr 2fr;
+    justify-content: space-between;
+    align-items: center;
+    overflow: hidden;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    left: 0;
+    z-index: 1;
+    background-color: var(--secondary);
+    padding: 0.6rem 0rem;
+  }
+  
+  .navbar a{
+      text-decoration: none;
+      color: var(--primary);
+  }
+  
+  .navbar .top-links a{
+    color: var(--font);
+  }
+  
+  .navbar .top-links .fa-heart:hover{
+    color: var(--fav);
+  }
+  
+  .navbar .top-links .fa-shopping-cart:hover{
+    color: var(--blue-text);
+  }
+
+  .nav-cart:hover{
+    color: var(--blue-text);
+  }
+
+  .nav-heart:hover{
+    color: var(--fav);
+  }
+  
+  /* nav btn ends */
+  .logo{
+      margin-left: 2rem;
+      text-shadow: 1px 1px 4px  var(--primary);
+      letter-spacing: 0.2rem;
+  }
+  
+  
+  /* input navbar */
+  
+  
+  
+  .top-links {
+    display: flex;
+    justify-content:flex-end;
+    align-items:center;
+    margin-right: 2rem;
+  }
+  .child-ecom {
+    margin: 1rem;
+    position: relative;
+    font-size: large;
+  }
+  
+  .badgeecom {
+    border-radius: 50%;
+    position: absolute;
+    top: -0.5rem;
+    left: 0.7rem;
+    font-size: small;
+    background-color: var(--red);
+    font-weight: bold;
+    color: var(--other-secondary);
+  }
+  
+  .one {
+    padding: 0.05rem 0.2rem;
+  }
+  .two {
+    padding: 0.01rem 0.35rem;
+  }
+  
+  i:hover{
+      text-shadow: 1px 1px 4px  var(--primary);
+      cursor: pointer;
+  }
+  
+  /* nav btn */
+  .btn{
+      padding: 0.4rem 1.2rem;
+      cursor: pointer;
+      border-radius: 1rem;
+      background-color: var(--buttons);
+      border: none;
+      font-weight: bold;
+  }
+  .btn:hover{
+      box-shadow: 1px 1px 4px 2px var(--primary);
+  }
+  
+  
+  
+  .input-field{
+      padding: 0.6rem 1.8rem;
+      border-radius: 0.8rem;
```

Comment:
border radius should be in px,  bcz like we don't want it to change according to screen size, so its bad practice to give border radius in rem vikram sir told about this in one of his sessions.

Reply:
got it, will change